CraftOrdersLib/data/scripts/player/map/mapcommands.lua

```diff
@@ -3,299 +3,343 @@
 -- MIT License 2019
 
 local moddedMapCommands = {}
+local windows = {}
 
 function MapCommands.registerModdedMapCommand(id, commandInfo)
-    if moddedMapCommands[id] == nil then
-        moddedMapCommands[id] = commandInfo
-    end
+	if moddedMapCommands[id] == nil then
```

Comment:
if you set your spacing to match the original file, the diff will be a lot cleaner. Also, I think the lack of extra indentation inside the `if onClient() then` block is intentional.

also adding ?w=1 ignores whitespace change, but it's good form to match the original style.